js/app/modules/card/defaultFamily.js

```diff
@@ -192,21 +291,33 @@ const Thumb = new Module.Class({
     vfunc_size_allocate: function (alloc) {
         this._layout.clear_constraints();
```

Comment:
Actually, talking to @ebassi here (and mixed with what we already spoke), the most efficient way would be to 1) create ALL constraints at _init (once) and then 2) enable/disable groups of constraints (the API allow this).